File: leetcode/2161.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        ls, es, ms = [], [], []

        for n in nums:
            if n < pivot:
                ls.append(n)
            elif n == pivot:
                es.append(n)
            else:
                ms.append(n)

        return ls + es + ms

File: leetcode/2161.kt
class Solution {
    fun pivotArray(nums: IntArray, pivot: Int): IntArray {
        val less = nums.filter({ it < pivot })
        val equal = nums.filter({ it == pivot })
        val greater = nums.filter({ it > pivot })
        return (less + equal + greater).toIntArray()
    }
}
